pkg/orchestrator/pod.go

```diff
@@ -385,20 +385,6 @@ func fillPodResourceVersion(p *model.Pod) error {
 	return nil
 }
 
-// mapToTags converts a map for which both keys and values are strings to a
-// slice of strings containing those key-value pairs under the "key:value" form.
-func mapToTags(m map[string]string) []string {
```

Comment:
In the first version I tried to move this to another place `orchestrator/util` but due to binary sizing and build constraints it is better to keep it here for now